client/store/localCart.js

```diff
@@ -0,0 +1,68 @@
+const cartState = {
+  cart: JSON.parse(localStorage.getItem('cart') || '[]')
+}
+
+//still need to add delete item from cart type,creator, and thunk. Might be easier once I have access to the cart page itself in order to make sure the button is operational.
+
+//action type
+
+const SET_CART = 'SET_CART'
+const GET_CART = 'GET_CART'
+
+//action creator
+
+const setCart = item => {
+  return {
+    type: SET_CART,
+    item
+  }
+}
+
+const getCart = cart => {
+  return {
+    type: GET_CART,
+    cart
+  }
+}
+
+//thunk
+export function _setCart(item) {
+  return dispatch => {
+    try {
+      item.quantity = 1
+      cartState.cart.push(item)
+      let localCart = JSON.stringify(cartState.cart)
```

Comment:
see initial comment in this file - cartState.cart would become simpler here without the cart property